classes/game.rb

```diff
@@ -0,0 +1,17 @@
+class Game
+  # Attributes
+  attr_accessor :id, :name, :genre, :multiplayer, :last_played_at
+
+  # Methods
+  def initialize(id, name, genre, multiplayer, last_played_at)
+    @id = id
+    @name = name
+    @genre = genre
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
```

Comment:
Kindly remove @id, @name and @genre instance variables on lines 8 and 9.
I'm asking this because @label and @genre were recently removed from Item class. And @id variable is created by Item class and its constructor is already implemented there.

Since Game class inherits from Item class, it should pass attributes to its superclass like this:
`super(publish_date, archived: archived)`
You will also need to provide these attributes in the constructor:
`def initialize(multiplayer, last_played_at, publish_date)` or `def initialize(multiplayer, last_played_at, publish_date, archived: false)`